code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -625,4 +625,61 @@
 	id = "drunkenblumpkin"
 	description = "A weird mix of whiskey and blumpkin juice."
 	color = "#1EA0FF" // rgb: 102, 67, 0
-	boozepwr = 35
\ No newline at end of file
+	boozepwr = 35
+
+/datum/reagent/consumable/ethanol/lava_lamp
+	name = "Lava Lamp"
+	id = "lava_lamp"
+	description = "Orange juice/vodka mixture combined with tomato juice at an extremely high temperature. Mild alcohol content."
+	color = rgb(251, 102, 30)
+	boozepwr = 5 //Not typically consumed for its strength
+	var/temperature_modifier = 1 //Deals less damage based upon how hot it is
+
+/datum/reagent/consumable/ethanol/lava_lamp/reaction_mob(mob/living/M, method)
+	if(iscarbon(M))
+		if(method == INGEST) //Breathe in and out really fast to make the liquid colder
+			M.audible_message("<span class='danger'>Holy shit, that's hot!</span>", "<span class='warning'>[M] hyperventilates!</span>")
+			temperature_modifier = 0.7
+		if(method in list(TOUCH, PATCH, VAPOR)) //Direct contact with skin
+			M.audible_message("<span class='danger'>Ow ow ow! That stuff <i>hurts</i>!</span>", "<span class='warning'>[M] yelps in pain!</span>")
+		if(method == INJECT) //In the bloodstream
+			M.visible_message("<span class='warning'>[M]'s skin begins to emit a slight glow!</span>", "<span class='boldannounce'>OW OW OWWW</span>")
+			M.emote("scream")
+			temperature_modifier = 1.3 //It's melting your fucking body from the inside
+	..()
+
+/datum/reagent/consumable/ethanol/lava_lamp/on_mob_life(mob/living/M)
+	M.adjustFireLoss((volume / 15) * temperature_modifier) //Volume-dependent burn damage affected by temperature; for instance, 50u would deal around 3 burn
+	return ..() || .
+
+/datum/reagent/consumable/ethanol/bluepc
+	name = "Bluepc" //A jab at people who scream "AI MALF" when they see an APC that's currently gaining charge
```

Comment:
you should just straight up name it "Blue APC" so that people can go "BLUE APC IN THE BAR" (kinda like the Syndicate Bomb drink)